web/main/views.py

```diff
@@ -568,8 +566,10 @@ class SectionTOCView(APIView):
     @method_decorator(user_has_perm("casebook", "viewable_by"))
     @method_decorator(user_has_perm("section", "viewable_by"))
     def get(self, request, casebook, section, format=None):
-        section.content_tree__load()
-        return Response(SectionOutlineSerializer(section).data)
+        mscq = manually_serialize_content_query(
+            ContentNode.objects.filter(id=section.id) | section.contents
```

Comment:
I don't quite understand this line; I think I'm stumbling over the `|`. Can you add a comment?